src/OrchardCore.Modules/OrchardCore.Setup/README.md

```diff
@@ -15,3 +15,18 @@ During setup, all recipes have access to the setup screen values using these par
 | `DatabaseTablePrefix` | The database table prefix. |
 
 These parameters can be used in the recipe using a scripted value like `[js: parameters('AdminUsername')]`.
+
+## Configuration
+`OrchardCore.Setup` can be configured through `appsettings.json` as the following:
```

Comment:
https://english.stackexchange.com/questions/68376/as-follows-vs-as-the-following